java-sdk/src/main/java/org/sagebionetworks/bridge/sdk/models/studies/StudySummary.java

```diff
@@ -0,0 +1,68 @@
+package org.sagebionetworks.bridge.sdk.models.studies;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import java.util.Objects;
+
+import static org.sagebionetworks.bridge.sdk.utils.BridgeUtils.TO_STRING_STYLE;
+
+/**
+ * Represent a study summary object
+ */
+@JsonDeserialize(as=StudySummary.class)
+public class StudySummary {
+    private String name;
+    private String identifier;
+    private String type;
+
+    public String getName() {
+        return this.name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getIdentifier() {
+        return this.identifier;
+    }
+
+    public void setIdentifier(String identifier) {
+        this.identifier = identifier;
+    }
+
+    public String getType() {
+        return this.type;
+    }
```

Comment:
Do other model objects have a type field? I don't believe so and you can remove this in these hand-created model objects.